src/app/core/nav-menu/components/license/license.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'cs-license',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="license">
+      {{ 'NAVIGATION_SIDEBAR.LICENSE.LICENSE_1' | translate: { year: getCurrentYear() } }}
+      <a href="https://bitworks.software/" target="_blank">
```

Comment:
Links with `taget="_blank"` should have `rel="nofollow noopener"` because of security issues. Check this [article](https://habr.com/post/282880/) for details.